main.css

```diff
@@ -34,13 +35,57 @@ body {
   color: hsl(218, 54%, 20%);
 }
 
-.header nav .navigation {
-  display: none;
+.navigation {
+  position: fixed;
+  left: -100%;
+  top: 70px;
+  gap: 0;
+  display: flex;
+  flex-direction: column;
+  background-color: hsl(0, 0%, 100%);
+  width: 100%;
+  height: 100vh;
+  justify-content: flex-start;
+  transition: 0.3s;
+}
+
+.header .navSec nav .navigation .nav-item {
+  width: 85%;
+  height: 50px;
+  list-style: none;
+  margin-left: 50px;
+}
+
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design
  - Make sure to make the welcome text invisible when the mobile menu appears to match the Figma design.
  - Make the close button size and design match the Figma design.
  - Make sure the size of the navigation links matches the Figma design.

| Your Design |Figma Design |
| :---:   | :---: | 
| ![Screenshot from 2023-01-31 08-45-52](https://user-images.githubusercontent.com/92303891/215711762-38ba9032-981f-4698-a97a-ad5b3cc69a90.png)| ![Screenshot from 2023-01-31 08-36-17](https://user-images.githubusercontent.com/92303891/215711841-3d094937-0829-4da3-bc73-b5d8c9cb35b5.png)|